sdk/identity/identity/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
```

Comment:
@bterlson I had to change the target back to `es5` because the Rollup bundle was being emitted with `class` keywords in it and that wasn't working with the Webpack build in the project that consumes this library.  This now causes the error-checking tests for IdentityClient to fail because the AuthenticationError subclass of Error can't be compared correctly anymore.

Do you know if I can target `es6` when building for the browser, or if there's another way to make the error type comparable again while using the `es5` target?  Happy to chat offline tomorrow if you're around.

Reply:
Which project is using webpack to consume this? Webpack should support class keyword so it might be a misconfiguration. You should not need to target es5 for browsers, es6 should be sufficient for all supported browser versions.

Reply:
It's an example browser use project I'm creating.  Since you say Webpack should support it, I'll take a closer look at my Webpack config to figure out what I'm doing wrong.

Reply:
Actually it wasn't Webpack that caused the issue, it's building `identity` itself with Rollup.  Here's the error I get from the browser build:

```
dist-esm/src/index.js → browser/identity.min.js...
   96 |      * the specific failure.
   97 |      */
>  98 |     class AuthenticationError extends Error {
      |          ^ Unexpected token: name «AuthenticationError», expected: punc «;»
   99 |         constructor(statusCode, errorBody) {
  100 |             let errorResponse = {
  101 |                 error: "unknown",
```

Reply:
Need to replace rollup-plugin-uglify with rollup-plugin-terser.

Reply:
That did the trick, thanks a bunch!